### PR TITLE
added candidate_count for Vertex models

### PR DESCRIPTION
**Pull request by lkuligin**
- **Description:**  added support for `candidate_count` parameter on Vertex

### CODE REVIEW

**Comment by lkuligin** [comment]
@baskaryan take a look when you have time, please

File: libs/langchain/langchain/chat_models/vertexai.py [hunk 1]
```diff
@@ -177,16 +174,22 @@ def _generate(
 
         question = _get_question(messages)
         history = _parse_chat_history(messages[:-1])
-        params = self._prepare_params(stop=stop, **kwargs)
+        params = self._prepare_params(stop=stop, stream=should_stream, **kwargs)
```
Comment (baskaryan) [inline libs/langchain/langchain/chat_models/vertexai.py:177]: if should_stream is true we would have already returned in L173

Comment (lkuligin) [reply libs/langchain/langchain/chat_models/vertexai.py]: agree, fixed.

**Comment by lkuligin** [comment]
@baskaryan take another look at you convinience, please

**Comment by baskaryan** [comment]
lgtm, thank you @lkuligin!

**Pull request merged**